webapp/src/components/pages/AboutUs.vue

```diff
@@ -1,21 +1,25 @@
 <template>
   <div class="space-y-8 mb-8">
-    <h1 class="text-disco-cyan font-disco text-3xl text-center">The <DISCO/> team</h1>
+    <h1 class="text-disco-cyan font-disco text-3xl text-center">
+      The <DISCO /> team
+    </h1>
 
     <IconCard>
       <template #title> About Us </template>
-      <p><DISCO/> is the result of a collaboration between two research labs:</p>
+      <p>
+        <DISCO /> is the result of a collaboration between two research labs:
+      </p>
 
       <ul class="list-disc list-inside">
         <li>
           <a
-          class="text-blue-500 hover:underline"
-          href="https://www.epfl.ch/labs/mlo/"
-          target="_blank"
+            class="text-blue-500 hover:underline"
+            href="https://www.epfl.ch/labs/mlo/"
+            target="_blank"
           >
-            MLO
+            <span>MLO</span>
```

Comment:
why the span? it is a single word and no specific style is applied

```suggestion
            MLO
```

Reply:
Otherwise the line breaks introduce whitespace around the word (" MLO "), another fix is to remove the line break but I find it less legible: 
```
<a
href>"..."
>MLO</a>
```
Do you have a preference?

Reply:
On second thoughts I'll go with remove line breaks, EPFL and AriadneLabs are already this way and it's not that bad